### DISCUSSION TITLE
I got student status on github, but copilot still asks for me to pay

### DISCUSSION THREAD

**Discussion by JustMeNova** [community/community#44901] [Copilot Conversations]
### Select Topic Area

Bug

### Body

So a few month ago i submitted everything to get github copilot as a student, but I believe because i had the 60 day free trial, it did not activate for free and still asks for payment, so id like to know how to activate it for free now. 

Thanks

**Comment by samuel-ma** [comment]
@JustMeNova 
Accepting the 60 day trial makes u an immediate paid subscriber after the end of the trial [learn more here](https://docs.github.com/en/copilot/quickstart)
You can contact github support [here](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket)

**Comment by iamcjwilliams** [comment]
Sorry for the trouble here, please visit [here](https://education.github.com/discount_requests/pack_application) in order to signup for free student access.
